### ISSUE TITLE
fastq-dump parallel error in tutorial

### ISSUE THREAD

**Issue by Koen-vdl** [ISUgenomics/bioinformatics-workbook#58]
Thanks for this great resource. fastqc dump is fairly poorly documented so this workbook was a great starting point for me. 

I think there is a small error on: https://bioinformaticsworkbook.org/dataAcquisition/fileTransfer/sra.html#gsc.tab=0

`parallel --jobs 3 "fastq-dump --split-files --origfmt --gzip {}" ::: SRR.numbers`

I believe this will only work if you `cat  ` SRR.numbers

`parallel --jobs 3 "fastq-dump --split-files --origfmt --gzip {}" ::: $( cat SRR.numbers)`

**Comment by molecules** [comment]
Yes, $( cat SRR.numbers) would work, or you can also use four colons instead of three (as just now suggested in pull request #60).
